department-of-veterans-affairs/caseflow
# Test special issues flow

Resolves #6854  

### Description
This PR finishes the special issue flow.

1) Inserts it into the AMA flow, but not the legacy flow.
1) Removes non-compensation special issues
1) Adds a test

### Testing Plan
1. Log in as BVASCASPER and chose the first case (it's an AMA case)
1. Go through the draft decision checkout flow
1. Ensure the first page is a special issues page.
1. Select things, hit continue.
1. Cancel and redo the flow ensure the same boxes are checked.
